config/config.go

```diff
@@ -265,6 +265,27 @@ type DeploymentConfig struct {
 // defaults to the namespace configured for IstioNamespace (which itself defaults to 'istio-system').
 type IstioComponentNamespaces map[string]string
 
+
+// Threshold configuration
+type Threshold struct {
+	Namespace string `yaml:"namespace"`
+	ThresholdAlerts    []ThresholdAlert  `yaml:"threshold_alerts"`
+}
+
+type ThresholdAlert struct {
+	Rule       string   `yaml:"rule"`
+	Kind       string   `yaml:"kind"`
+	Metric     string   `yaml:"metric"`
+	Filter     ThresholdFilter `yaml:"filter,omitempty"`
+	Expression string   `yaml:"exp"`
+	Alert      string   `yaml:"alert"`
```

Comment:
May you document in comments how to use these fields ?
That would help me to review better the tests.